mlir/test/Dialect/Transform/test-interpreter.mlir

```diff
@@ -2109,3 +2109,48 @@ transform.sequence failures(propagate) {
   transform.yield 
 }
 }
+
+// -----
+
```

Comment:
This doesn't really belong to `test-interpreter`, which, as the name indicates, is supposed to test the interpreter itself. Please start a new file.